lib/ansible/modules/cloud/docker/docker_login.py

```diff
@@ -316,7 +449,7 @@ def main():
         registry_url=dict(type='str', default=DEFAULT_DOCKER_REGISTRY, aliases=['registry', 'url']),
         username=dict(type='str'),
         password=dict(type='str', no_log=True),
-        email=dict(type='str'),
+        email=dict(type='str', removedin_version='2.14'),
```

Comment:
should be `removed_in_version`, see https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#removed-in-version

Reply:
Thanks for spotting this, I've just fixed it here. For `docker_image`, I'll create a PR with some more text there :)